assets/css/map/controls/_map_button.scss

```diff
@@ -0,0 +1,30 @@
+@use "../../color/tokens_2024" as tokens;
+
+.c-map-button {
+  box-shadow: 0 0 0 2px rgb($black, 0.15);
+  --bs-btn-font-size: 0.625rem;
+  --bs-btn-border-radius: 0.2rem;
+  --bs-btn-color: #{$black};
+  --bs-btn-bg: #{$white};
+  --bs-btn-hover-color: #{$black};
+  --bs-btn-hover-bg: #{tokens.$eggplant-50};
+  --bs-btn-active-color: #{tokens.$eggplant-500};
+  --bs-btn-active-bg: #{$white};
+  --bs-btn-focus-box-shadow: 0 0 0 2px #{rgb(tokens.$eggplant-500, 0.5)};
+  --bs-btn-disabled-color: #{tokens.$gray-700};
+  --bs-btn-disabled-bg: #{tokens.$gray-100};
+  --bs-btn-disabled-opacity: 1;
```

Comment:
note(non-blocking): Just so we're on the same page, what I'm mulling over, is whether we should be using css variables here or scss variables here. And additionally, whether tokens is the right level to pull from.

I'm not sure if I have an opinion with any good reasons yet though.

Reply:
But, I'm okay with this being approved and merged since It Works™️

Reply:
> note(non-blocking): Just so we're on the same page, what I'm mulling over, is whether we should be using css variables here or scss variables here. And additionally, whether tokens is the right level to pull from.
> 
> I'm not sure if I have an opinion with any good reasons yet though.

Basically, yes - though I'm not sure if scss variables would really work because I'm not sure how we would localize the change to just the map button as opposed to all buttons. I was mainly wondering if there was some other option that I had missed, or if it would be better to just write our own CSS for this. But, if you're okay with this I think it works well so I'll go ahead and merge!

Reply:
Oh the scss variables vs css variables I was referring to was
```
  --bs-btn-hover-bg: #{tokens.$eggplant-50};
```
and something like this instead
```
  --bs-btn-hover-bg: var(--color-eggplant-50);
```


Part of me wishes `tokens.$eggplant-50 === "var(--color-eggplant-50)"` so we didn't even have to think about it and we'd get the "Good DX™️" for "free"